README.md

```diff
@@ -8,38 +8,28 @@ Yawoen company has hired you to implement a Data API.
 
 **1 - Load company data in a database**
 
-Read data from CSV file and load into the database to create a catalog named **companies**.
+Read data from CSV file and load into the database to create an entity named **companies**.
 
 This catalog should contain the following fields: id, company name and zip code. 
 
 suport file: q1_catalog.csv
 
 
-**2 - API to integrate data using a database**
+**2 - An API to integrate data using a database**
 
-Yawoen company started to capture website data and want to integrate with your database.
+Yawoen company started to capture website data from another source and want to integrate it with the entity you've just created on the database.
 
-This new source data has the following protocol:
+This new source data has the following input protocol:
 
-- Response format: JSON
+- Input file format: CSV
 
 - Parameters:
 
     - Name: string
     - Zip: string 
     - Website: string
 
-Example: 
-
-```
-{
-	"name": "Yawoen Business Solutions",
-	"zip":"10023",
-	"website": "www.yawoen.com"
-}
```

Comment:
Not happy with the example?

Reply:
it is an output example and we are talking about the input